pkg/question/question.go

```diff
@@ -121,6 +121,13 @@ func AskQuestion(input *AskQuestionInput) string {
 
 		// No match at all
 		fmt.Println("Input invalid. Please try again.")
+		if input.DefaultOptionRepr != nil {
+			fmt.Printf("%s [%s]:  ", input.QuestionString, *input.DefaultOptionRepr)
+		} else if input.DefaultOption != nil {
+			fmt.Printf("%s [%s]:  ", input.QuestionString, *input.DefaultOption)
+		} else {
+			fmt.Printf(input.QuestionString + ": ")
+		}
```

Comment:
Since lines 124-130 here are the same as 63-69, can we extract this logic into a new function that we invoke in these two spots instead? So that we don't repeat the same logic, and avoid changing one but not the other in the future, among other things :)

Reply:
Thanks for the feedback. Will add a new function and invoke it.